sdk-setup/src/sdk-manage

```diff
@@ -1469,6 +1469,7 @@ synchronise_target() {
 # Qt5 qml imports are here
 + $libdir/qt5/qml**
 + /usr/include**
++ $libdir/*/include**
```

Comment:
What about the /usr/share dirs which were mentioned in the bug?

Reply:
Those from cmake are already covered, those from libtool are not relevant IIUC.

Reply:
Actually, cmake is *not* covered as far as I can tell: /usr/lib/cmake is synced, but /usr/share/cmake/Modules and /usr/share/cmake/Modules and /usr/share/cmake/include, which were listed in the bug, are not.

Reply:
Indeed :( and it's the commit on the very bottom... aargh

Reply:
I will push another commit directly to master after merging all the PRs.

Reply:
https://github.com/sailfishos/sdk-setup/commit/57ec7b54c62522b42b2257be16ad6f45d2f494ae